src/api/index.js

```diff
@@ -113,10 +113,12 @@ export function getPipeline({ name, namespace }) {
   return get(uri);
 }
 
-export function getPipelineRuns({ namespace, pipelineName } = {}) {
+export function getPipelineRuns({ namespace, pipelineName, filters } = {}) {
   let queryParams;
+  const pipelineNameArray = [`tekton.dev/pipeline=${pipelineName}`];
+  queryParams = { labelSelector: filters };
   if (pipelineName) {
-    queryParams = { labelSelector: `tekton.dev/pipeline=${pipelineName}` };
+    queryParams = { labelSelector: filters.concat(pipelineNameArray) };
```

Comment:
remove line 118

`queryParams.labelSelector.push(`tekton.dev/pipeline=${pipelineName}`)`

Is filters ever undefined ? Also down stream if filters is empty will we end up with trailing `?` on our api calls